src/map/instance.cpp

```diff
@@ -239,6 +239,9 @@ void CInstance::ClearEntities()
         if (static_cast<CBattleEntity*>(entity.second)->isAlive())
         {
             entity.second->PAI->ClearStateStack();
+
+            if (entity.second->objtype == TYPE_MOB)
```

Comment:
what scenario did you encounter where this was needed?  I think the purpose of it was simply to make things stop attacking you when you complete the instance (and get put into a cutscene immediately)

Reply:
i believe it was a cheap fix for the cs not triggering when char was in the middle of a ws or casting, canceling player action

Reply:
also, but not 100% sure anymore, i think it sometimes also happend that mobs kept on attacking or reaggroed very quickly

Reply:
hmm.. let's remove it for now, and then put up a separate issue for it when you've confirmed it